src/DataService/NotificationListener.php

```diff
@@ -52,11 +56,15 @@ class NotificationListener
     /**
      * NotificationListener constructor.
      *
-     * @param string                            $clientId A string to identify this client in debugging.
+     * This method always uses the read-only version of a service.
+     *
+     * @see https://docs.theplatform.com/help/wsf-subscribing- to -change-notifications#tp-toc2
```

Comment:
Huh, I wonder what happened to this link. Will fix.

Reply:
Don't forget to fix it!